db/seeds.rb

```diff
@@ -114,7 +114,8 @@
                              description: "Gooooood",
                              price: 8.99,
                              status: "retired",
-                             categories: [appetizer_cat])
+                             categories: [appetizer_cat],
+                             image: File.open('public/corn_dog.jpg'))
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.